src/components/JournalEntryCard/JournalEntryCard.tsx

```diff
@@ -25,7 +25,7 @@ const JournalEntryCard: React.FC<JournalEntryCardProps> = (props) => {
 
   const dismissSnackBar = () => {
     setSnackBarDetails({ ...snackBarDetails, show: false });
-};
+  };
```

Comment:
Need to look into linter?? I believe for formatting the files and creating consistent organizational styling